tests/tools/test_io.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+import pytest
+
+from ssz.examples.zoo import (
+    Zoo,
+    zoo,
+)
+
+
+@pytest.mark.parametrize(
+    "int_as", (
+        "int",
+        "str",
+        "hex",
+    )
+)
+def test_import_export(tmpdir, int_as):
+    path = tmpdir / "zoo.dump"
+    with open(path, "w") as f:
+        json.dump(zoo.to_formatted_dict(int_as=int_as), f)
```

Comment:
Not sure if writing it to a file is necessary here, `json.dumps` seems sufficient.